build.gradle.kts

```diff
@@ -16,6 +16,7 @@ plugins {
     id("io.gitlab.arturbosch.detekt") version "1.13.1"
     // ktlint linter - read more: https://github.com/JLLeitschuh/ktlint-gradle
     id("org.jlleitschuh.gradle.ktlint") version "9.4.0"
+    id("groovy")
```

Comment:
I'm not super happy about adding groovy, but it's okay ;)

Reply:
I want to avoid Groovy in production code. afaik, the plugin will not add the libs to production code, only support the tests

Reply:
But still, the tests would be written in a different language than the code they would test.

Reply:
afaik, you can now write spock tests in Groovy or JUnit Tests in Kotlin